lte/gateway/c/session_manager/datastore/CMakeLists.txt

```diff
@@ -22,8 +22,8 @@ list(APPEND PROTO_SRCS "")
 list(APPEND PROTO_HDRS "")
 
 set(ASYNC_ORC8R_CPP_PROTOS common redis)
-set(ASYNC_LTE_CPP_PROTOS session_manager policydb mobilityd apn subscriberdb)
-set(ASYNC_LTE_GRPC_PROTOS session_manager)
+set(ASYNC_LTE_CPP_PROTOS "")
```

Comment:
i think this might be fixed on master :)

Reply:
This time I have not touched this CMakeLists.txt file